projects/canopy/src/lib/side-nav/docs/guide.stories.mdx

```diff
@@ -8,12 +8,12 @@ import Feedback from '../../docs/common/feedback.md?raw';
 
 ## Usage
 
-Import the side nav module in your application:
+Import the side nav components in your application:
 
 ```js
 @NgModule({
   ...
-  imports: [ ..., LgSideNavModule ],
+  imports: [ ..., LgSideNavComponent, etc. ],
```

Comment:
Since all the components are now standalone they all need to be imported. Paul and I agreed that we can use etc. to indicate that all components related / required need to be added there